examples/Example/src/prot_calculator/index.tsx

```diff
@@ -0,0 +1,354 @@
+  }
+
+  function open() {
+    drag.value = withTiming(-props.offset.value, { duration: 300 });
+
+    setOpened(true);
+  }
+
+  function close() {
+    drag.value = withTiming(0, { duration: 300 });
+
+    setOpened(false);
+  }
+
+  const translationStyle = useAnimatedStyle(() => {
+    return {
+      transform: [{ translateY: props.offset.value + drag.value }],
+    };
+  });
+
+  const dragGesture = useAnimatedGesture(
+    Gesture.pan()
+      .setOnUpdate((e) => {
+        'worklet';
+        let value = drag.value + e.changeY;
+
+        if (value > -props.offset.value) {
+          drag.value = -props.offset.value;
+        } else if (value < 0) {
+          drag.value = 0;
+        } else {
+          drag.value = value;
+        }
+      })
+      .setOnEnd((e, s) => {
+        'worklet';
+        let value = drag.value + e.changeY;
+
+        if (opened) {
+          if (value < -props.offset.value - 100) runOnJS(close)();
+          else runOnJS(open)();
+        } else {
+          if (value > 100) runOnJS(open)();
+          else runOnJS(close)();
+        }
+      })
+  );
+
+  scrollView.current?.scrollToEnd({ animated: true });
+
+  return (
+    <GestureMonitor gesture={dragGesture}>
+      <Animated.View
+        style={[styles.output, translationStyle]}
+        onLayout={measure}>
+        <ScrollView
+          ref={(ref) => {
+            if (!opened && ref) ref.scrollToEnd({ animated: false });
+            scrollView.current = ref;
+          }}
+          enabled={opened}
+          contentContainerStyle={{ flexGrow: 1 }}>
+          <View style={{ flexGrow: 1 }} />
+
+          {props.history.map((exp, index) => {
+            return <Expression expression={exp} key={index} />;
+          })}
+
+          <Expression expression={props.expression} />
+        </ScrollView>
+        <View style={styles.handleView}>
+          <View style={styles.handle} />
+        </View>
+      </Animated.View>
+    </GestureMonitor>
+  );
+}
+
+function Expression(props) {
+  return (
+    <View style={styles.expression}>
+      <View style={{ flexDirection: 'column' }}>
+        <Text style={styles.expressionText}>{props.expression}</Text>
+        <Text style={styles.expressionResult}>{props.expression}</Text>
+      </View>
+    </View>
+  );
+}
+
+function Input(props) {
+  const translationStyle = useAnimatedStyle(() => {
+    return {
+      transform: [{ translateY: props.offset.value }],
+    };
+  });
+
+  function append(symbol) {
+    if (symbol == '<') {
+      props.setHistory((h) => [...h, props.expression]);
+      props.setExpression((e) => '');
+    } else {
+      props.setExpression((e) => e + symbol);
+    }
+  }
+
+  return (
+    <Animated.View
+      style={[styles.input, translationStyle]}
+      onLayout={props.measure}>
+      <NumPad append={append} />
+      <Operations append={append} />
+    </Animated.View>
+  );
+}
+
+function NumPad(props) {
+  let buttons = [7, 8, 9, 4, 5, 6, 1, 2, 3, '<', 0, '.'];
```

Comment:
```suggestion
  const buttons = [7, 8, 9, 4, 5, 6, 1, 2, 3, '<', 0, '.'];
```

Reply:
Fixed in 51547a54fcab702fb339c9af3f141c33b0d06de7